### PR TITLE
feat: Circle mode mobile version

### PR DESCRIPTION
**Pull request by SiqueiraD**
This way, we can add circles using the mobile web app.

### CODE REVIEW

File: src/modes/circle/circle.mode.ts [hunk 1]
```diff
@@ -122,6 +123,8 @@ export class TerraDrawCircleMode extends TerraDrawBaseDrawMode<CirclePolygonStyl
 			this.clickCount++;
 			this.setDrawing();
 		} else {
+			if (document.documentElement.classList.contains("is-mobile"))
```
Comment (JamesLMilner) [inline src/modes/circle/circle.mode.ts:126]: This line is too specific to the mapping library you're using - we need a more agnostic way to approach this! Let's have a conversation about how that might work in #106

Comment (SiqueiraD) [reply src/modes/circle/circle.mode.ts:126]: I got this information from the file main.js
https://github.com/JamesLMilner/terra-draw/blob/02937d041669ee1899768ac9fdfd197b15ac8622/docs/assets/main.js#L1999

Comment (JamesLMilner) [reply src/modes/circle/circle.mode.ts:126]: So this is the from the generated docs file, not from the source or distribution file of the library. We don't apply this class anywhere with the actual library code itself.

**Comment by JamesLMilner** [comment]
Hey, just to say thanks again for your interest - as I alluded to in my previous comment I'm not sure this is the right approach for this issue. I've set up an issue for discussion on this problem where we can discuss possible solutions. I'm going to close this for now but I'm more than happy to take another look at a PR If we come to a conclusion on how to fix this.

**Pull request closed without merging**